src/SandayTasks/Records/ClassRoom.java

```diff
@@ -0,0 +1,6 @@
+package SandayTasks.Records;
+
+public record ClassRoom(int grade, int classNum, Student[] students) {
```

Comment:
```suggestion
public record Classroom(int grade, int classNum, Student[] students) {
```
classroom is one word